syncstorage-spanner/src/manager/session.rs

```diff
@@ -64,6 +64,8 @@ pub struct SpannerSessionSettings {
     pub max_lifespan: Option<u32>,
     /// Max idle time of a Session
     pub max_idle: Option<u32>,
+    /// Interval for sweeper task releasing unused connections.
+    pub pool_max_idle: Option<u32>,
```

Comment:
We already have a `database_pool_connection_max_idle` setting, let's use that instead -- the pool implementation uses it to timeout connections as it checks them out, but I believe the sweeper's going to do a better job of reducing the number than that code

Reply:
You bet @pjenvey , I wasn't 100% sure about reusing some of those settings so opted to just make a new one. Can I also pull this out of the Option and set a default?